core/src/main/res/values/colors.xml

```diff
@@ -12,6 +12,9 @@
   <color name="blue800">#1565c0</color>
   <color name="pink300">#f06292</color>
   <color name="actionModeBackground">#4285F4</color>
+  <color name="gray_700">#616161</color>
```

Comment:
we shouldn't set text color to this specific tone, now it can't change for night mode or if you do you specify in the `values-night` resource directory `gray_700>#something that is most certainly not 700`

You should specify colors for their intention like the other attributes in the theme `colorOnSurface/colorError` and then set these colors via another color depending on day/night.

On the topic of these colors specifically I am also these days more likely to design for OLED screens because they are more beautiful so I tend towards using black/white where I can.

All this said good job, it does look nice.